backoffice/templates/backoffice/user.html

```diff
@@ -63,6 +63,11 @@
                   {% endif %}</p>
               </div>
 
+              <div class="col-sm-5"></div>
+              <div class="col-sm-7">
+                  <p><a href="{% url 'backoffice:impersonate-user' userform.instance.username %}">{% trans "Impersonate" %}</a></p>
```

Comment:
Manque la traduction, je propose "posséder"

![exorciste](http://idata.over-blog.com/4/68/33/42/dossier-1.0/the-exorcist.73209.jpg)